### DISCUSSION TITLE
Min Height not working

### DISCUSSION THREAD

**Discussion by SaraKaterji** [mpdf/mpdf#1711] [General]
Hello,

I am trying to use mpdf to generate a pdf document and I would love to add a paragraph element <p> with min-height. I added the following:
```
.field-p{
    min-height:500px;
}
and
<p class="field-p">{{$data}}</p>

```
It is not taking the min-height into consideration.

I am using the latest version of laravel framework (9), php8 and the latest version of mpdf (included in the laravel wrapper provided by [carlos](https://packagist.org/packages/carlos-meneses/laravel-mpdf))

Any ideas?

**Comment by finwe** [comment]
Yes, min-height is not supported for p element https://mpdf.github.io/css-stylesheets/supported-css.html
